.github/workflows/CD.yml

```diff
@@ -0,0 +1,116 @@
+name: CD
+
+on:
+  push:
+    tags:        
+      - '**' 
+
+jobs:
+  validate-tag:
+    name: Validate tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+      
+      - name: Check no jumps in tags
```

Comment:
Also, does this account for when there is no earlier tag defined?

Reply:
Yes, that's what line 23 does:
```
if [[ $(git tag -l | wc -l) != 1 ]]; then # If there is only one tag, no need to check
```
If only one tag is found it means there were no earlier tags defined, and the check is not performed.